cypress/integration/Modal.spec.ts

```diff
@@ -103,6 +105,10 @@ describe('Modal', () => {
           });
 
           it('should trap focus inside the modal element', () => {
+            h.modal
+              .get()
+              .pipe(h.modal.getCloseButton)
+              .should('have.focus');
```

Comment:
This is tested in the previous test, why is it tested again?

Reply:
because of the timeout, we have to first wait for the focus to be moved to closing button.

Reply:
Ah. This is a wait.

Reply:
I debated adding this explicit wait in the `beforeEach`, but decided against that. We should either add a comment that this group is to wait for focus transfer, or extract to a function called `waitForFocusTransfer`